### DISCUSSION TITLE
[Bug]: Switching Proxy is complex if you have more than one server (more than one domain)

### DISCUSSION THREAD

**Discussion by takeerdevs** [coollabsio/coolify#3178] [Improvement Requests]
### Description

If you've just installed Coolify, I can switch proxy from traefik to Caddy after i stop proxy.. then i can switch proxy to Caddy. What if there could be a drop down on Server creation to select which proxy you would like to use.

The issue comes if you've added one server ( a domain) then server created uses traefik by default then you've to switch it to Caddy. If you add another server (another domain), gets created using traefik, which you'll need to stop it and switch to Caddy, but seems there is an issue switching the second server's proxy which does remove initial proxy (Caddy) and there is a weird behaviour on getting the second domain work using Caddy as proxy.

### Minimal Reproduction (if possible, example repository)

Create one server and link to a domain, then switch proxy to Caddy then add another server (domain) then switch to proxy to Caddy.

### Exception or Error

Couldn't see error as sometimes all proxies stopped working and couldn't reach Coolify dashboard, but there seems to be an auto proxy deployment when it is not reachable

### Version

v4.0.0-beta.276

**Comment by maxinux** [comment]
Confirmed, it would seem to be something you would do for an entire resource  stack or server rather than server by server, at least should set a default for future servers
